docs/install-on-minikube.md

```diff
@@ -0,0 +1,70 @@
+# Install on Azure AKS
+
+## Pre-requisites
+- [Minikube](https://github.com/kubernetes/minikube)
+- To install etcd using these instructions, you must have [Helm](https://helm.sh) in your client environment.
+- [Download the Compose on Kubernetes installer](https://github.com/docker/compose-on-kubernetes/releases)
+
+## Create compose namespace
+
+- First you should have the confirmation that everything is running and `kubectl` is pointing to the minikube vm by running `minikube status`.
+If not, please start it by running `minikube start`. That should create and/or start minikube's vm.
+- Then, create a compose namespace by running `kubectl create namespace compose`
+
+## Deploy etcd
+
+If you already have an etcd instance, skip this.
+
+### Install Helm server side component

```

Comment:
If this is common with the AKS install; it might make sense to factor it out. WDYT @silvin-lubecki @simonferquel

Reply:
+1 as it will be easier to maintain.